webapp/publisher/snaps/views.py

```diff
@@ -876,8 +876,8 @@ def get_register_name_dispute():
 @login_required
 def post_register_name_dispute():
     try:
-        snap_name = flask.request.form.get("snap-name")
-        claim_comment = flask.request.form.get("claim-comment")
+        snap_name = flask.request.form.get("snap-name", "")
+        claim_comment = flask.request.form.get("claim-comment", "")
```

Comment:
What does this have to do with bleach on listing page?

Reply:
we sanitize what is sent to the store